packages/components/app/styles/components/code-block/index.scss

```diff
@@ -0,0 +1,209 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// CODE-BLOCK
+//
+
+
+// Note: "theme" contains just color variables and syntax highlighting styles
+@import "theme";
+
+// DIMENSIONS
+$code-block-line-numbers-width: 49px; // 3em = 49px
```

Comment:
I had this set at a static width in Figma because if number of lines was less than 10 (i.e. only had one digit), the width would change which seemed like an inconsistency. I'm not saying this is the wrong solution, I'm only noting the reasoning why the Figma component was created in a more static manner.

Reply:
Prism.js uses a static width in the pre-built themes. (They had this value with a comment: `width: 3em; /* works for line-numbers below 1000 lines */`) It doesn't adjust the width based on the number of lines so it's most pragmatic to design it with a width that looks ok with the longest realistically expected line number. We can do more testing around this...

I think it should use rems or ems ideally (for end users who customize their browser default font/font-size) but we use px a lot elsewhere so maybe fine to keep as px...